administering-tugboat-crew/index.md

```diff
@@ -0,0 +1,129 @@
+- [Remove a user from a project](#remove-a-user-from-a-project)
+- [Change user permissions](#change-user-permissions)
+- [Add a Tugboat bot to your team](#add-a-tugboat-bot-to-your-team)
+- [User permission levels explained](#user-permission-levels-explained)
+
+> #### Info::Permissions and organizing projects
+>
+> User permissions in Tugboat are handled on a project basis - not based on the
+> repository. If you've got discrete sets of users who should not be able to
+> access a specific repo; for example, you've got two repos, one containing an
+> internal 'top secret' project, and one containing a front-facing static site
+> that an external vendor manages; you'll want to make those different projects.
+
+## Add a user to a project
+
+1. Click your username in the upper right-hand corner, and go to Projects;
+2. Click the **My Projects** link;
+3. Select the project where you want to add the user;
+4. Click the **Project Settings** link to the right of the project's title;
+5. In the **Invite a User to This Project** section, add the recipient's email
+   address, and select the appropriate
+   [user type](#user-permission-levels-explained) from the drop-down;
+6. Press the big blue **Invite** button!
+
+The user you've invited will get an email from `support@tugboat.qa` with a link
+to accept the invitation. If the user doesn't see the invite:
+
+- Have them check Inboxes and Spam for this email address, or;
+- **Copy Link** and share it another way from the Pending Invites section of the
+  Project Settings, or;
+- Hit the **Re-send** link from the Pending Invites section of the Project
+  Settings.
+
+![Copy Link or Re-Send from Pending Invites](_images/add_user_pending_invite.png)
+
+## Remove a user from a project
+
+1. Click your username in the upper right-hand corner, and go to Projects;
+2. Click the **My Projects** link;
+3. Select the project where you want to remove the user;
+4. Click the **Project Settings** link to the right of the project's title;
+5. In the **Manage Users** section, look for the user you want to remove, and
+   click the Remove link.
+
+## Change user permissions
+
+1. Click your username in the upper right-hand corner, and go to Projects;
+2. Click the **My Projects** link;
+3. Select the project where you want to remove the user;
+4. Click the **Project Settings** link to the right of the project's title;
+5. In the **Manage Users** section, look for the user whose permissions you want
+   to change, and select the appropriate
+   [user type](#user-permission-levels-explained) from the **Access** drop-down.
+
+## Add a Tugboat bot to your team
+
+By default, comments that Tugboat posts to the linked git provider account
+display as the user whose credentials link the account to Tugboat. If you
+instead want those comments to show as coming from Tugboat, you can add a
+Tugboat bot to your team:
+
+1. Create an account called "Tugboat Bot" at your preferred git provider; i.e.
```

Comment:
Let's not dictate what to call the bot account. The providers require unique account names anyway, so as soon as one person creates "Tugboat Bot" nobody else can. I think we can get away with just saying "Create a tugboat bot account at..." and our target audience will understand.